draft-ietf-trans-rfc6962-bis.md

```diff
@@ -868,7 +868,11 @@ chain that the log used to verify the submission.
 Each log is identified by an OID, which is one of the log's parameters (see
 {{log_parameters}}) and which MUST NOT be used to identify any other log. A
 log's operator MUST either allocate the OID themselves or request an OID from
-the Log ID Registry (see {{log_id_registry}}). Various data structures include
+the Log ID Registry (see {{log_id_registry}}).
+(Recall that OID's are do not require a central registration, although
```

Comment:
Typo/Suggestion: s/Recall that OID's are do not/OID allocations do not/

Reply:
took the suggestion.  Also added a sentence. Fixup commit pushed.